content/docs/command-reference/add.md

```diff
@@ -139,32 +143,24 @@ $ tree
 Let's check the `data.xml.dvc` file inside:
 
 ```yaml
-md5: aae37d74224b05178153acd94e15956b
 outs:
-  - cache: true
-    md5: d8acabbfd4ee51c95da5d7628c7ef74b
-    metric: false
-    path: data.xml
-meta: # Special field to contain arbitary user data
-  name: John
-  email: john@xyz.com
+  - md5: 6137cde4893c59f76f005a8123d8e8e6
+    path: data.xml # .dvc files support comments :)
 ```
 
-This is a standard `.dvc` file with only one output (in the `outs` field). The
-hash value should correspond to a file path in the <abbr>cache</abbr>.
+This is a standard `.dvc` file with only one output (`outs` field). The hash
+value (`md5` field) corresponds to a file path in the <abbr>cache</abbr>.
 
-> Note that the `meta` values above were entered manually for this example. Meta
-> values and `#` comments are not preserved when a `.dvc` file is overwritten
+> Note that `#` comments are not preserved when a `.dvc` file is overwritten
 > with the `dvc add`, `dvc run`, `dvc import`, or `dvc import-url` commands.
```

Comment:
Should we just remove all the info. about YAML comments from this doc? Or at least try to move it to another example and not the first one?

Reply:
where does this question come from? did we change any logic?

Reply:
There's no longer `meta` AFAIK. I've been removing that.

As for comments, those are still supported but explained in the DVC file guide. I'm saying the add cmd ref may not be the best place to even mention them, especially not in the most basic example.

Reply:
no meta only for .dvc files, right?

Reply:
I decided to remove it in 9bd9962b. It's explained in the files guide, which is linked to several times form this doc.